### ISSUE TITLE
[Bug][A11Y] Filter chip component should have a toggle role instead of a button

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#956]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14+

### OS Version

iOS 18.5

### App version

0.19.0

### App build number

1757669301

### App build type

alpha

### Describe the issue

The filter chip component is used as a checkbox, to filter some datas.
The button role is not relevant for that use.

### Expected Behavior

Filter chip should act as a checkbox/switch, so it should have a Toggle role

### Accessibility settings

Voice Over

**Comment by pylapp** [comment]
@pya35 Quel est ton retour suite à cette évolution soumise via _build alpha_ ? Si tu ne 'las pas reçue dis moi :)

**Comment by pya35** [comment]
@B3nz01d @pylapp on va rester sur le role bouton dans le cas générique. Toutefois, est-il prévu des variantes du rôle sur le _filter chip_ selon le contexte comme un radio button ou checkbox ?

**Comment by pylapp** [comment]
@pya35  Je ne pense pas.
